resources/SwitchMap vs MergeMap.md

```diff
@@ -0,0 +1,349 @@
+
+```javascript
+|------d$------d$------c$------d$------c$--------->
+       |
+       |-🐶-🐶-🐶-🐶-🐶-🐶-🐶-🐶-🐶-🐶-🐶-🐶-🐶>
+       		    |
+       		    |-🐶-🐶-🐶-🐶-🐶-🐶-🐶-🐶-🐶-🐶->
+       		          |
+       		          |-🐱-🐱-🐱-🐱-🐱-🐱-🐱-🐱->
+       		                  |
+       		                  |-🐶-🐶-🐶-🐶-🐶-🐶>
+       		                         |
+       		                         |-🐱-🐱-🐱-🐱>
+       
+```
+AnimalCloner is out of control. What we want is _one_ stream of animals for each time an animal enters the cloner. Instead we end up with _multiple_ streams. `Switch` to the rescue, but first we need to make sure we understand why there are multiple streams here.
+
+`dogClones$`: This is a stream of dogs that emit every 100th of a second
+
+`catClones$`: This is a stream of cats that emit every 100th of a second
+
+`catsAndDogs$`: This is a stream that emits a mix of **dogClones$** and **catClones$** every second. The important take-away here is that this is a **stream of streams**.  That's why when we pass `catsAndDogs$` into `AnimalCloner()` we get that nasty barage of streams. What we want is one stream.  And this is exactly what's missing from the **switch** description:
+
+>Switch operates on a stream where each emission is another stream.  Switch will   stop listening to the original stream at each emission and change to the new emitted stream.  At _**every subsequent emission from the root stream, it will stop listening to the previous child stream**_.  
+
+Let's add `switch()`.
+
+```javascript
+const dogClones$ = Rx.Observable.interval(100).mapTo('🐶');
+const catClones$ = Rx.Observable.interval(100).mapTo('🐱');
+
+const catsAndDogs$ = Rx.Observable.from([
+  dogClones$,
+  dogClones$,
+  catClones$,
+  dogClones$,
+  catClones$,
+]).concatMap(animals$ => Rx.Observable.of(animals$).delay(1000));
+
+const AnimalCloner = (animals$) => animals$;
+
+AnimalCloner(catsAndDogs$).switch();
+```
+
+<a target="_blank" href="https://rxviz.com/v/38MYm38v">Run this code</a>
+
+Fixed. Now it's just one nice stream that _switches_ to the latest dog or cat stream over time. 
+
+```javascript
+|------d$------d$------c$------d$------c$--------->
+       |
+       |-🐶-🐶-🐶-🐶-🐱-🐱-🐱-🐶-🐶-🐶--🐱-🐱-🐱>
+```
+Update the verbs with our switch definition:
+
+| Verb     | What it does  |
+| -------- |-------------| 
+| Map      | Use the _**value of something**_ to change it to _**something else**_.  | |
+| Merge    | Merge takes two or more observable streams and _**combines**_ them together.      | 
+| Switch   | Switch operates on a stream where each emission is another stream.  Switch will   stop listening to the original stream at each emission and changes to the new emitted stream.  At every subsequent emission from the root stream, it will stop listening to the previous child stream.       | 
+
+### MergeMap & SwitchMap
+Now that we have all the verbs defined, we can combine the definitions together to understand these two operators. 
```

Comment:
There might be some value in splitting this into two files - one that helps define these map verbs, and another to compare merge/switch